### DISCUSSION TITLE
How to Split Pane?

### DISCUSSION THREAD

**Discussion by kmaustral** [oxdc/obsidian-vertical-tabs#60] [Q&A] [answered]
The Split Pane command puts a copy of the note into a new pane. With Vertical Tabs, this just puts a new copy of the note in the sidebar. How can you put a copy of the note in a new pane with Vertical Tabs enabled?

**Comment by oxdc** [comment, marked as answer]
Hi! You may have enabled the Zen Mode, which would hide inactive splits (panes). Please disable it to see other splits. I recorded a video about how to split the screen and hide & focus with Zen Mode:


https://github.com/user-attachments/assets/5c0299f6-59aa-4514-a764-c76b25df27a4

**Comment by kmaustral** [comment]
Yes, you're right! I'm glad I can keep trying out this promising plugin.

**Comment by kmaustral** [comment]
Can I confirm that you can't have tabs in a group that are in different panes? If that's not possible now, could it be a feature request?

**Reply by oxdc** [reply]
@kmaustral Sorry, I am afraid that I didn't fully understand. Could you please provide some annotated screenshots of your desired outcome? That would be really helpful! Thank you!

**Comment by kmaustral** [comment]
![Untitled design](https://github.com/user-attachments/assets/6da0707d-36ca-4c04-899d-66b71fe800cd)

So when you split the screen, the second note ends up in a different group. Is there a way to make it in the same group?

**Reply by oxdc** [reply]
> Is there a way to make it in the same group?

Definitely yes! Simply drag the note (2024-11-07) and drop it in the desired spot! Note that you have enabled automatic tab sorting, which will reorder the tabs as you demand after dropping.

**Reply by oxdc** [reply]
Here's a demo video for this:

1. You can drag and drop the vertical tab and place it in a desired group;
2. You can drag and drop the vertical tab to the "new group" button to create a new group and place it there;
3. You can drag and drop the native horizontal tab to anywhere you want just as before using *Vertical Tabs*.

https://github.com/user-attachments/assets/21aba43e-9ff9-4ea3-ac3c-a8f45b49b85a

**Comment by kmaustral** [comment]
Thank you for the time you've taken to explain this. What I am asking about is whether it is possible to have tabs from different panes in the same group. Do groups have to be arranged in panes? 

I appreciate the complexity of coding all this. If this arrangement is hard-wired into the plugin, I'll find a workaround.

**Comment by oxdc** [comment]
> Do groups have to be arranged in panes?

Yes, this design is intended to make *Vertical Tabs* in sync with Obsidian's native UI.

> If this arrangement is hard-wired into the plugin, I'll find a workaround.

I'm afraid that, in the near future, this is necessary.

> whether it is possible to have tabs from different panes in the same group

There's already a FR for sections in groups! Check it out [here](https://github.com/oxdc/obsidian-vertical-tabs/discussions/62#discussioncomment-11173307). I'm working on it and will take some time.

For your request "have tabs from different panes in the same group", I think the best solution would be "Workspaces", which provides further structures. Since the build-in Workspace plugin does not work well, I'll implement a better one in *Vertical Tabs*.

- Top Level: Session (like Workspace)
- Group (split or pane)
- Section (further splitting within groups)

**Comment by Azmoinal** [comment]
> - Top Level: Session (like Workspace)
> 
> - Group (split or pane)
> 
> - Section (further splitting within groups)

My vote for this kind of feature. I like the idea to have some sessions to switch between with a specific name for each one.

Honestly speaking I didn't like workspace plugin into obsidian and it doesn't work on mobile.

For my use case, I didn't need to switch between Obsidian panes configurations but just between some sets of tabs related to the argument on which I should work during the day.
